packages/core/src/V.ts

```diff
@@ -61,9 +62,9 @@ const ignoreValidator = new IgnoreValidator(),
   nullOrUndefinedValidator = new IsNullOrUndefinedValidator(),
   notEmptyValidator = new NotEmptyValidator(),
   notBlankValidator = new NotBlankValidator(),
-  emptyToNullValidator = new ValueMapper(val => (isNullOrUndefined(val) || val === '' ? null : val)),
-  emptyToUndefinedValidator = new ValueMapper(val => (isNullOrUndefined(val) || val === '' ? undefined : val)),
-  undefinedToNullValidator = new ValueMapper(val => (val === undefined ? null : val)),
+  emptyToNullValidator = new ValueMapper((val) => (isNullOrUndefined(val) || val === '' ? null : val)),
+  emptyToUndefinedValidator = new ValueMapper((val) => (isNullOrUndefined(val) || val === '' ? undefined : val)),
+  undefinedToNullValidator = new ValueMapper((val) => (val === undefined ? null : val)),
```

Comment:
Could it be that the parentheses are added due to testing Prettier rules? Tested locally, and the lines weren't modified.

Reply:
Yeah, some of those slipped by before I did that `"arrowParens": "avoid"` configuration. Will fix.